bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+## Ferroamp Binding
+
+The Ferroamp binding is used to get live data from Ferroamp EnergyHub
+
+The Ferroamp binding is compatible with EnergyHub Wall and EnergyHub XL, and use connection to your local EnergyHub via LAN or similar.
+Data and commands are received/sent using MQTT where the user connects to the MQTT broker residing on the EnergyHub.
+
+Contact Ferroamp support to enable MQTT in the EnergyHub and to get the Username and Password:
+
+https://ferroamp.com/om-ferroamp/
+
+## Supported Things
+
+The binding retrieves data from the different parts of the Ferroamp EnergyHub such as:
+
+- `Ehub`: EnergyHub Wall and EnergyHub XL.
+- `Sso`: Solar string optimizer.
+- `Eso`: Bidirectional DC/DC converter for connection of battery.
+- `Esm`: Energy Storage Module.
+
+## Discovery
+
+Discovery is not supported.
+
+## Thing Configuration
+
+The following configuration-parameters are available.
+
+| Name            | Type    | Description                              | Default | Required | Advanced |
+|-----------------|---------|------------------------------------------|---------|----------|----------|
+| hostName        | text    | Hostname or IP address of the device     | N/A     | yes      | no       |
+| userName        | text    | Username to access the device            | N/A     | yes      | no       |
+| password        | text    | Password to access the device            | N/A     | yes      | no       |
+| hasBattery      | boolean | Is there a battery connected or not?     | N/A     | no       | yes      |
+| ssoS0           | boolean | Is there a first SSO unit connected?     | N/A     | no       | no       |
+| ssoS1           | boolean | Is there a second SSO unit connected?    | N/A     | no       | no       |
+| ssoS2           | boolean | Is there a third SSO unit connected?     | N/A     | no       | no       |
+| ssoS3           | boolean | Is there a fourth SSO unit connected?    | N/A     | no       | no       |
+
+| The unique serial number is marked on the side of the SSO unit. Ex. PS00990-A04-S20120476
+
+## Channels
+
+| Channel Type ID                       | Item Type                  | Read/Write | Label                                    | Description 
+|----------------------------------------------------------------------------------------------------------------------------------------------
+| grid-frequency                        | Number:Frequency           | R          | Estimated Grid Frequency                 | Estimated Grid Frequency |
+| ace-current-l1                        | Number:ElectricCurrent     | R          | ACE Current L1                           | ACE equalization current set-points in Arms |
```

Comment:
What is ACE and what is Arms ?

Reply:
I don't have this kind of system myself, ( I have a Sonnen Battery 10 Performance) so I have to relay on the documentation as:
Phase balancing, also called ACE (Adaptive Current Equalization), means that EnergyHub moves current between the three incoming phases in the property so that the consumed current in the property is evenly distributed between the phases. It thus counteracts phase imbalances such as when one of the phases is more heavily loaded as a result of many electrical components being installed on the same phase. 
The phase balancing works by the EnergyHub
Draws current from the phases that have a lower load
Converts alternating current to direct current in order to move it
Converts direct current back to alternating current
Outputs alternating current on the high load phase
and
this is far away from my knowledge, but regared to Google so is it:
Arms, Amps root mean square, seen in electrical specifications often as the unit of current, Arms.
and in some Ferroamp documentation is it described as Unit.

Reply:
Check, better to adapt description similar to SSO:

`` 
Adaptive Current Equalization (ACE) equalization current set-points in Amps root mean square (Arms)
```

Reply:
Changed in README